packages/database/src/migrations/20210124013906-FixLaosSchoolsDevelopmentPartnerSupportMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const MAP_OVERLAY_IDS = [
+  'Laos_Schools_Major_Dev_Partner_Province',
+  'Laos_Schools_Major_Dev_Partner_District',
+];
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    update "mapOverlay" mo
+    set "measureBuilderConfig" = regexp_replace(mo."measureBuilderConfig"::text, '"valueToMatch": "Yes"','"condition": 1','g')::jsonb
```

Comment:
These overlays actually have the wrong config so this is just to fix an existing bug. To apply conditions on entityAggregation, we need to use property `condition` instead of `valueToMatch`

Reply:
Another great catch